specs/core/0_beacon-chain.md

```diff
@@ -789,6 +796,17 @@ def compute_domain(domain_type: DomainType, fork_version: Version=Version()) ->
     return Domain(domain_type + fork_version)
 ```
 
+### `compute_domain_wrapper_root`
+
+```python
+def compute_domain_wrapper_root(object: SSZObject, domain: Domain) -> Root:
```

Comment:
`object` is a keyword in python. Probably best to not overwrite it here.

how about `ssz_object`?